stdlib/importlib/metadata/_meta.pyi

```diff
@@ -12,11 +13,11 @@ class PackageMetadata(Protocol):
     @property
     def json(self) -> dict[str, str | list[str]]: ...
 
+# Protocol deviates from runtime implementation.  The latter is not assignable
```

Comment:
Could you add test cases to show that this Protocol is indeed compatible with types like `pathlib.Path`?

https://github.com/python/typeshed/tree/main/test_cases